youtube_dl/extractor/tubetugraz.py

```diff
@@ -348,4 +349,5 @@ def _gen_format_id(self, format, type, transport, format_types):
         else:
             pre_index = ""
 
-        format["format_id"] = "%s-%s-%s%d" % (type, transport, pre_index, index)
+        format_id = "%s-%s-%s%d" % (type, transport, pre_index, index)
+        format["format_id"] = format_id
```

Comment:
What does this change do ?

Reply:
This change makes the line fit into the <80 character limit (it was exactly 80 characters long).

Reply:
<80 character limit is a soft limit; you are allowed to over 80 characters limit since it doesn't make it hard to read